helm-chart/renku-notebooks/templates/cronjob.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
-  name: {{ template "notebooks.fullname" . }}-remove-user-registry-secrets
+  name: {{ include "notebooks.fullname" . | trunc 23 | trimSuffix "-" }}-remove-user-registry-secrets
```

```suggestion
  name: {{ include "notebooks.fullname" . | trunc 30 | trimSuffix "-" }}-cull-registry-secrets
```
Just to have a somewhat smaller probability for mutilated naming....